src/lambdas/update_rules_processor/index.py

```
@@ -173,29 +172,30 @@ def lambda_handler(event, context):
 
         try:
             test_manifest(df, pattern_list)
-
+        except AssertionError:
+            LOGGER.error("Exception: Manifest test failed")
```

thinking about it - this exception is kind of vague. I don't yet understand what it means as Im not sure what the purpose of the test is (havent looked at the test yet)

---

Not sure if it was you or Nick who mentioned https://snyk.io/blog/the-dangers-of-assert-in-python/

(asserts can be silently ignored in "production" environments!)

---

Ah okay, a message like that would be a bit clearer

I will make a ticket to clean this up and change it to be a validation function and fix up any error message there